src/Day02.kt

```diff
@@ -18,15 +9,10 @@ fun main() {
 
     // Part 1
     fun part1(input: List<String>): Int {
+        var totalPaper = 0
 
-        val measurements = input
-            .map { it.split('x') }
-            .map { it.map { it.toInt() }}
-
-        var totalPaper   = 0
-
-        for (i in measurements.indices) {
-            val (length, width, height) = Triple(measurements[i][0], measurements[i][1], measurements[i][2])
+        for (dimensions in input) {
+            val (length, width, height) = dimensions.split("x").map { it.toInt() }
```

Comment:
Notice that I'm only using one call to `map` here. You had a bit of redundancy, I think.

Reply:
But definitely the right idea!

Reply:
And notice the use of destructuring. Since the call to `split` then `map` results in a new list of three `Int` values we can use a destructring assignment (i.e. the `val (length, width, height)` bit) to assign each of those new values to named variables right away. 

The use of `Triple` basically accomplished the same thing, but Kotlin supports this without the need for `Triple`.